<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>781</width>
                        <height>531</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="windowVLayout">
                    <item>
                        <layout class="QHBoxLayout" name="termHLayout">
                            <item>
                                <widget class="QLabel" name="termLabel">
                                    <property name="text">
                                        <string>Term:</string>
                                    </property>
                                    <property name="buddy">
                                        <cstring>termLineEdit</cstring>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="termLineEdit"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="QLabel" name="definitionLabel">
                            <property name="text">
                                <string>Definition:</string>
                            </property>
                            <property name="buddy">
                                <cstring>definitionTextEdit</cstring>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QTextEdit" name="definitionTextEdit">
                            <property name="acceptRichText">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="addTermButton">
                            <property name="text">
                                <string>Add Term</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="termListLabel">
                            <property name="text">
                                <string>List of Terms:</string>
                            </property>
                            <property name="buddy">
                                <cstring>termList</cstring>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QListWidget" name="termList"/>
                    </item>
                    <item>
                        <widget class="QPushButton" name="exportTermsButton">
                            <property name="text">
                                <string>Export Terms</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QToolBar" name="mainToolBar">
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <tabstops>
        <tabstop>termLineEdit</tabstop>
        <tabstop>definitionTextEdit</tabstop>
        <tabstop>addTermButton</tabstop>
        <tabstop>termList</tabstop>
        <tabstop>exportTermsButton</tabstop>
    </tabstops>
    <resources/>
    <connections>
        <connection>
            <sender>addTermButton</sender>
            <signal>clicked()</signal>
            <receiver>termList</receiver>
            <slot>update()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>400</x>
                    <y>315</y>
                </hint>
                <hint type="destinationlabel">
                    <x>400</x>
                    <y>452</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>termList</sender>
            <signal>itemDoubleClicked(QListWidgetItem*)</signal>
            <receiver>termLineEdit</receiver>
            <slot>update()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>400</x>
                    <y>452</y>
                </hint>
                <hint type="destinationlabel">
                    <x>422</x>
                    <y>63</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
